unattended_installer/install_functions/checks.sh

```diff
@@ -330,4 +351,15 @@ function checks_ports() {
         exit 1
     fi
 
-}
\ No newline at end of file
+}
+
+# Checks if the first version is greater equal than to second one
+function check_versions() {
+
+    if test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" == "$1"; then
```

Comment:
This seems to return 0 when they are equal and 1 if they are not, shouldn't it be `-ge`?

Reply:
Not exactly. The command lists the function arguments, put them into separate lines, and orders them in reverse order (descending order). So, if the first argument is the version at the top of the list, means that is greater or equal to the second. 

Besides, versions can not be compared with conditional operators like strings or integers.

Reply:
Great, very ingenious, good job